log/config.go

```diff
@@ -43,6 +47,16 @@ func (v verboseMode) Apply(logger Logger) error {
 	return logger.(settableVerbosity).SetVerbose(v.on)
 }
 
+func NewWriter(w io.Writer) Config {
```

Comment:
`NewWriter` vs `SetOutput`
thoughts?

Reply:
IMO, New implies the option is creating something for you, whereas this method is taking your writer and applying it, I would probably prefer `SetOutput`